doc/nrf/releases/release-notes-1.8.0.rst

```diff
@@ -0,0 +1,832 @@
+
+* Bug fixes:
+
+  * Fixed an issue that was causing the HID keyboard LEDs to remain turned on after host disconnection while no other hosts were connected.
+  * Fixed an issue that was causing an assertion failure in the :ref:`nrf_desktop_hid_state` on the nRF Desktop peripheral device during the boot of the host device connected through USB.
+
+nRF Pelion Client
+-----------------
+
+* Updated:
+
+  * Updated to use DTS overlays instead of Kconfig configuration files for setting up external flash memory.
+  * The application switched to using the configuration file scheme based on :file:`prj.conf` files.
+    See :ref:`gs_modifying_build_types` for details.
+
+Thingy:53: Matter weather station
+---------------------------------
+
+* Added:
+
+  * Support for the Power Source cluster, used to expose information about the battery.
+  * Support for the Identify cluster, which uses the built-in buzzer to help find the device.
+
+* Updated:
+
+  * Updated to use DTS overlays instead of Kconfig configuration files for setting up external flash memory.
+  * The application switched to using the configuration file scheme based on :file:`prj.conf` files.
+    See :ref:`gs_modifying_build_types` for details.
+
+nRF Machine Learning
+--------------------
+
+* Updated:
+
+  * Updated to use DTS overlays instead of Kconfig configuration files for setting up external flash memory.
+
+nRF9160: Serial LTE modem
+-------------------------
+
+* Updated the ``#XFOTA`` command to accept an integer parameter to specify the PDN ID to be used for the download, instead of the APN name.
+* Added new AT commands related to the General Purpose Input/Output (GPIO).
+* Added the ``#XUUID`` command to read out the device UUID from the modem.
+* Added to the ``XNRFCLOUD`` command the following features:
+
+  * The possibility to send to and receive from nRF Cloud JSON messages in data mode.
+  * The ability to read out the ``sec_tag`` and the UUID of the device.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Bluetooth samples
+-----------------
+
+* Added :ref:`central_and_peripheral_hrs` sample.
+* Updated some samples to use DTS overlay instead of Kconfig for external flash.
+* Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+* :ref:`ble_llpm` sample - Added role selection.
+* :ref:`peripheral_uart` sample - The sample is now the default one for the :ref:`ble_rpc` library.
+  The sample runs out of the box with a serialized Bluetooth Host.
+* :ref:`peripheral_hids_mouse` and :ref:`central_uart` samples - These samples now come with the :ref:`ble_rpc_host` child image configuration overlay.
+  The overlay shows how to configure an application running a serialized Bluetooth Host.
+  These samples run out the box with the :ref:`ble_rpc` library.
```

Comment:
Please add:

- ":ref:`direct_test_mode` - isolated Timer usage in the workaround for Anomaly 172 for the nRF52840 device." per https://github.com/nrfconnect/sdk-nrf/pull/6142/
- ":ref:`direct_test_mode` - replaced busy wait with idle mode to reduce RF noise coming from accesses to Flash on devices which don't have instruction cache." per https://github.com/nrfconnect/sdk-nrf/pull/6015
- "Removed the pairing_confirm callback from the Bluetooth samples where it was incorrectly used (accepted pairing without user interaction)." per https://github.com/nrfconnect/sdk-nrf/pull/5919/